src/main/java/nl/novi/techiteasycontroller/Services/TelevisionService.java

```diff
@@ -0,0 +1,107 @@
+package nl.novi.techiteasycontroller.Services;
+
+import nl.novi.techiteasycontroller.Dto.Output.TelevisionOutputDto;
+import nl.novi.techiteasycontroller.Exceptions.DuplicateName;
+import nl.novi.techiteasycontroller.Exceptions.RecordNotFoundException;
+import nl.novi.techiteasycontroller.Model.Television;
+import nl.novi.techiteasycontroller.Repository.TelevisionRepository;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class TelevisionService {
```

Comment:
Ik mis nog methodes om een RemoteController, CiMocule of WallBracket aan een Television te koppelen.